### DISCUSSION TITLE
Can not get user was login, I have provider token (JWT). Can any body help me.

### DISCUSSION THREAD

**Discussion by hoannv2592** [realworld-apps/realworld#1541] [Community]
https://[api.realworld.io/api/user](https://api.realworld.io/api/user

`
{
    "status": "error",
    "message": "missing authorization credentials"
}

`

**Comment by geromegrignon** [comment]
Helo @hoannv2592, how are you sending the jwt token with the request?

**Comment by hoannv2592** [comment]
@geromegrignon I have added jwt to header.But still error.

I try with https://realworld-docs.netlify.app/docs/specs/frontend-specs/swagger for test.It same error

**Reply by geromegrignon** [reply]
It's working fine for me.

Are you using the expected format, as explained in Swagger?

<img width="588" alt="image" src="https://github.com/gothinkster/realworld/assets/32737308/d16d0e34-d64f-4e41-aa54-13ee36ff2e66">

If you do, provide technical details about the way you are defining the token.

**Reply by hoannv2592** [reply]
@geromegrignon, I follow with step:
1. Login with account (email, password).
2. Reload page.
Results:
![image](https://github.com/gothinkster/realworld/assets/10227575/7887f24b-c9f6-4e3e-9022-7307966df103)
url :
https://react-redux-realworld.netlify.app/

**Reply by geromegrignon** [reply]
It sounds like there are some bug with the react redux implementation, you can open a bug issue in the repo: https://github.com/khaledosman/react-redux-realworld-example-app

But about Swagger it works fine for me.
How did you fill this part?

<img width="636" alt="image" src="https://github.com/gothinkster/realworld/assets/32737308/4f3e83dd-fc60-4f17-bd48-40d20b18a326">

**Reply by hoannv2592** [reply]
@geromegrignon Thank you.I miss prefix  string `Token` before `jwt`. Swagger it works fine.
